src/app/screens/ConfirmPayment/index.test.tsx

```diff
@@ -79,12 +83,16 @@ describe("ConfirmPayment", () => {
       );
     });
 
+    expect(screen.getByText("25 sats")).toBeInTheDocument();
```

Comment:
is this specific enough? because we have two fiat values in the document.

Reply:
Made sure the second value is really not there in the beginning.